Lecture6(GithubAPI)/components/AboutUsChildComponent.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Component } from 'react'
```

Comment:
If importing from same react, Instead of two lines do it in one. Like
import react, { component } from 'react'

But again no hard and fast rule. Both are right.

Reply:
yes thanks for suggestions!